src/components/ModelDialog/components/ModalContent/ModalContent.tsx

```diff
@@ -0,0 +1,40 @@
+import type { ReactNode } from 'react'
+
+import { StyledDescription, StyledFormFooter, StyledSection } from './styled'
+
+export type Props = {
+  onClose: () => void
+  onConfirm?: () => void
+  description?: string
+  isConfirmation?: boolean
+  children?: ReactNode
+}
+
+export const ModalContent = ({
+  description,
+  isConfirmation,
+  onClose,
+  onConfirm,
+  children,
+}: Props) => {
+  const descriptionText = description
+    ? description
+    : 'Er is geen back-up beschikbaar.'
```

Comment:
Why this default description?

Reply:
This was in the design as text. It is one of the most generic texts we have for a confirmation modal